docs/user-guide.rst

```diff
@@ -202,8 +202,13 @@ recommended to set the ``Content-Type`` header::
 Certificate verification
 ------------------------
 
+ .. note:: *New in version 1.25*
+
+    HTTPS connections are now verified by default
+    (``cert_reqs = 'CERT_REQUIRED'``) regardless of whether
+    CA certs are specified.
+
 It is highly recommended to always use SSL certificate verification.
-**By default, urllib3 does not verify HTTPS requests**.
 
 In order to enable verification you will need a set of root certificates. The easiest
```

Comment:
Is this still true?

Reply:
I don't know if this was ever completely true? You could set `cert_reqs` to `REQUIRED` and we'd load the default verification locations.

So now by default urllib3 tries to load the default certificate locations via [`SSLContext.load_default_certs()`](https://docs.python.org/3.7/library/ssl.html#ssl.SSLContext.load_default_certs) which apparently works well on Windows and on non-Windows uses [`SSLContext.set_default_verify_locations()`](https://docs.python.org/3.7/library/ssl.html#ssl.SSLContext.set_default_verify_paths) which is also reliable in my experience.

The only platforms that will have to use certifi or their own bundle are platforms that don't have `SSLContext.load_default_certs()` or don't define `SSLContext` at all because our implementation of `SSLContext` doesn't define `load_default_certs()` (Should our implementation try to implement this?).

As for `contrib/`: `PyOpenSSLContext` defines a functioning `load_default_certs()` and `SecureTransportContext` defines the method but does absolutely nothing with it so maybe it will fail cert verification unless SecureTransport loads certs by default? Will have to look at docs.

To sum it up, it's a mixed bag of whether we load system certs if no certificate locations are given. :)

Reply:
Then we should probs remove this, yeah?

Reply:
When you say this you mean the whole section on certifi? I'm not sure what your comment is directed at. If we remove that subsection then the section as a whole is pretty bare and the last part mentions system certificates as well. Maybe this whole section needs a rewrite?